deploy/examples/operator-openshift.yaml

```diff
@@ -248,6 +262,23 @@ data:
   #   - key: node.rook.io/cephfs
   #     operator: Exists
 
+  # (Optional) CephCSI NFS provisioner NodeAffinity(if specified, overrides CSI_PROVISIONER_NODE_AFFINITY).
+  # CSI_CEPHFS_PROVISIONER_NODE_AFFINITY: "role=nfs-node"
+  # (Optional) CephCSI NFS provisioner tolerations list(if specified, overrides CSI_PROVISIONER_TOLERATIONS).
```

Comment:
it will override `CSI_PROVISIONER_TOLERATIONS` if specified if not `CSI_PROVISIONER_TOLERATIONS` will be applied for nfs csi provisioner pod?

Reply:
> it will override `CSI_PROVISIONER_TOLERATIONS` if specified if not `CSI_PROVISIONER_TOLERATIONS` will be applied for nfs csi provisioner pod?

yes